cluster/gce/ubuntu/node.yaml

```diff
@@ -0,0 +1,281 @@
+From nobody Tue Aug 11 10:13:54 2015
```

Comment:
Generated by the tool https://github.com/number5/cloud-init/blob/master/tools/make-mime.py for packing multiple upstart jobs into one MIME file.